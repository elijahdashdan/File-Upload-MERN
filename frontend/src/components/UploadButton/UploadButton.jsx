import React from 'react';
import {Button} from 'react-bootstrap';

function UploadButton(props) {
    return (
        <div className="uploadButton">
        <Button variant="primary" size="lg" onClick={props.onClick}>Upload</Button>{' '}
        </div>
    )
}

export default UploadButton;