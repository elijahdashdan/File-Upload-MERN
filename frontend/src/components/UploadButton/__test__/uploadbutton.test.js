import React from 'react';
import ReactDOM from 'react-dom';
import UploadButton from '../UploadButton';
import renderer from 'react-test-renderer';

it("Renders without Crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<UploadButton />,div)
    ReactDOM.unmountComponentAtNode(div)
})

it('renders correctly', () => {
    const tree = renderer
      .create(<UploadButton />)
      .toJSON();
    expect(tree).toMatchSnapshot();
});