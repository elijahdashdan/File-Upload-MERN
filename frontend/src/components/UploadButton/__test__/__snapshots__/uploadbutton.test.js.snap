// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="uploadButton"
>
  <button
    className="btn btn-primary btn-lg"
    disabled={false}
    type="button"
  >
    Upload
  </button>
   
</div>
`;
